plugins/no_sir.py

```diff
@@ -0,0 +1,15 @@
+from errbot import BotPlugin
+
+
+class no_sir(BotPlugin):
+    """do not use sir to sound more friendly"""  # Ignore QuotesBear
+
+    def callback_message(self, msg):
```

Comment:
It's better to use `@re_botcmd`, check other modules for some useful patterns :wink: